_posts/2021/2021-09-16-5-things-i-learned-in-10-years.md

```diff
@@ -0,0 +1,124 @@
+At this point, emotional intelligence is very useful.
+You probably have heard somewhere this term before.
+But what does it mean you may ask?
+
+Basically, it consists of 4 parts:
+1. Perceiving of emotions: The ability to realize the emotions of people.
+This includes facial expressions, gestures, posture and voice.
+2. Using of emotions: The ability or knowledge to use emotions of yourself and other people, to ease cognitive activities, e.g. problem or conflict solving.
+3. Understanding: The ability to analyze emotions and how these can change over time.
+4. Managing: The ability to influence the emotions of yourself or other people.
+
+
+
+### 2. Think about 'The Big Picture'
+
+The big picture can have a lot of meanings, depending on the person you ask.
+While working on projects or products, usually you become the expert of your team's domain.
+A lot of developers tend to overestimate the importance of their own domain in the system.
+
+Let's assume there are 3 views on that:
+- Business: How to be more successful as a company and how can this be achieved?
+- User: How can we provide value to the end-user or customer with the code you write?
+- Technology: How to build software that is robust and lasts long (clean and homogeneous architecture, efficient, low memory usage etc.)?
+
+When you start working on new tasks, it is always helpful to have the answers to the what-why-how-questions.
+Usually, these questions helped me always to understand to get a grasp at the big picture:
+- have the end in mind (what)
+- understand and challenge the requirements (why)
+- connect the dots (how)
+
+
+### 3. Become a T-Shaped Person
+
+Many people argue if you should become a specialist or generalist in your industry.
+For me, the answer is a simple yes and no. But first, let us understand what T-Shaped means:
+
+> A T-shaped person has deep knowledge/skills in one area and a broad base of general supporting knowledge/skills.
+
+I always had the ambition to have a broad knowledge.
+It was not just my programming language and/or the framework I used to write the code in.
+I learned and tried to use new languages, concepts, new (web) technologies in simple pet projects.
+The opposite of a T-Shaped person is an I person with only deep knowledge in one field and no general knowledge.
+There is also the dash person with only general knowledge who has no deep knowledge of a topic.
+
+But why should you aim for that goal?
+
+- Collaboration with other people is easier as you have at least some basic knowledge about other topics
+- You stay interested in learning new things
+- You can break the ice easier with new colleagues
+- As you become more and more familiar with concepts outside of your knowledge, you become slowly immune against the [Paradox of Expertise](https://www.psychologytoday.com/us/blog/tracking-wonder/201906/the-paradox-expertise){:target="_blank"}
+- You will get tasks with more responsibility from your employer
```

Comment:
Da alle Unterpunkte in ganzen Sätzen formuliert sind, bitte jeweils am Ende noch einen Punkt ergänzen.